src/index.js

```diff
@@ -0,0 +1,3 @@
+import { getWeatherCurrent } from './APICaller';
+
+const data = await getWeatherCurrent('Vilnius');
```

Comment:
Probably preparation for doing other functionality, but don't fetch API like this. You should only fetch on demand from frontend. F.e. On place input change. On component loaded and etc.

Reply:
turi omeny, kad iš index.js necallint? nes galvoju, kad apart input changes, pirminis default/initial fetch veiks UI.js moduly, kur kokiam nors async displayWeatherData metodui papassinsiu šitą _data_ variable

Reply:
Jo, butent, kad is index.js necallint. Tu pasiruosei koda, kuri pasitestavai ir kuris veikia, bet jo nereiktu usint, kol neturi actual use case. Tai kai atsiras ui, kuriame atvaizduosi ta data, tada ir reiktu naudot